flow360/component/results/case_results.py

```diff
@@ -362,6 +412,24 @@ def as_dataframe(self):
 
         return pandas.DataFrame(self.values)
 
+    def wait(self, timeout_minutes=60):
```

Comment:
We have many wait functions. I remember we or the tidy 3D team made it a wrapper which can be used. I will add to tickets. We do not have to do it here.